### ISSUE TITLE
LFO waveform ghost aliasing

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/surge#4100]
![image](https://user-images.githubusercontent.com/2393720/111330876-9fc6d800-8670-11eb-80e1-3c1e14399687.png)

Still there ever since LFO ghost was implemented, but would be nice if it could be cleaned up for 1.9!

**Comment by baconpaul** [comment]
I wonder if it is the dotted line causing the problem. Maybe we should try a solid line with a non-255 alpha?

**Comment by mkruselj** [comment]
Ha maybe. But man I really like the dotted line. Maybe we can switch from dotted to solid after a certain Hz rate?

**Comment by baconpaul** [comment]
So I took a look. The dotted line isn't doing it. And Im not sure what is.

There is one micro-bug the second deactPath should be xc + 0.1 * not xc - 0.1* but that also doesn't fix it

I tried setting the line width to 1 and that didn't fix it

I printed the values going to the path and all are in the box. They are all in bounds. 

We know if we apply a clip rect we will be OK, but that's what blew up the oscillator draw, so we aren't safe to do that.

So I just don't know why vstgui path drawing is drawing out of bounds here.

ahh and typing this in made me figure it out.

Fix coming soon.
